code/modules/projectiles/guns/magic.dm

```diff
@@ -37,10 +36,14 @@
 	return charges
 
 /obj/item/gun/magic/newshot(params)
-	if(charges && chambered)
+	if(charges && chambered && !chambered.BB)
```

Comment:
Looks like you added the additional check in the end 😏

Reply:
Yes but it didn't solve the problem.

Reply:
Well yes